go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
-	github.com/verrazzano/verrazzano-monitoring-operator v0.0.0-20201110193547-32e100ccdb72
+	github.com/verrazzano/verrazzano-monitoring-operator v0.0.0-20210506131752-819e5ed265ec
```

Comment:
won't you be needing to change this _after_ the VMO changes go in?

Reply:
I suppose you could merge that one first

Reply:
I have it set to my VMO commit.  I can change it to the master VMO commit after I merge VMO

Reply:
Fixed to use VMO master commit